apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -894,9 +876,14 @@ func (api *ProvisionerAPI) processEachContainer(ctx context.Context, args params
 		return fmt.Errorf("cannot get container networking method: %w", err)
 	}
 
+	modelConfig, err := api.modelConfigService.ModelConfig(ctx)
+	if err != nil {
+		return fmt.Errorf("getting model config: %w", err)
```

Comment:
This error message lacks context.

Reply:
At this scale I don't think there's more context we can add. This function itself doesn't know why it's being called. Callers are responsible for adding that extra context.